app/src/main/res/layout/tracks_item.xml

```diff
@@ -8,6 +8,7 @@
         android:id="@+id/track_title"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
+        android:elevation="7dp"
```

Comment:
elevation attribute is not supported in Kitkat and Below. It will cause errors and possible crash.